grakn-engine/src/main/java/ai/grakn/engine/controller/CommitLogController.java

```diff
@@ -81,7 +78,7 @@ private String deleteConcepts(Request req, Response res){
         @ApiImplicitParam(name = COMMIT_LOG_COUNTING, value = "A Json Array types with new and removed instances", required = true, dataType = "string", paramType = "body")
     })
     private String submitConcepts(Request req, Response res) {
-        Keyspace keyspace = Keyspace.of(Optional.ofNullable(req.queryParams(KEYSPACE_PARAM)).orElse(defaultKeyspace));
+        Keyspace keyspace = Keyspace.of(req.queryParams(KEYSPACE_PARAM));
```

Comment:
I think there's a method `mandatoryQueryParam` that will throw if it's not there - which is what we need here.